generators/index.js

```diff
@@ -9,19 +9,12 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const excshell = require('@adobe/generator-app-excshell')
```

Comment:
That's about cleaning up some dead code right ?

Reply:
yes, everything goes through the Template Registry now for extensions